pkg/controller/argocd/argocd_controller.go

```diff
@@ -118,58 +93,35 @@ func filterPredicate(assert func(namespace, name string) bool) predicate.Funcs {
 	}
 }
 
-func assertArgoCD(namespace, name string) bool {
-	return namespace == argocdNS && argocdInstanceName == name
-}
-
-func assertArgoCDRoute(namespace, name string) bool {
+func argocdRouteFilter(namespace, name string) bool {
```

Comment:
`argoCDRouteFilter` for funciton name.

Reply:
I've updated the function name to `filterArgoCDRoute`